src/main/java/io/cdap/plugin/gcp/bigquery/sqlengine/BigQueryWrite.java

```diff
@@ -354,7 +354,7 @@ protected QueryJobConfiguration.Builder getInsertQueryJobBuilder(TableId sourceT
       .setCreateDisposition(JobInfo.CreateDisposition.CREATE_NEVER)
       .setWriteDisposition(JobInfo.WriteDisposition.WRITE_APPEND)
       .setPriority(sqlEngineConfig.getJobPriority())
-      .setLabels(BigQuerySQLEngineUtils.getJobTags("copy"));
+      .setLabels(BigQuerySQLEngineUtils.getJobTags("write"));
```

Comment:
I updated the BQ Write (BQ Sink Pushdown) labels to match the actual operation semantics (Write to the sink)